docs/learn/vadalog/02-language-primitives.md

```diff
@@ -167,24 +184,26 @@ manager(Y,X) :- employee(X).
 This ontology represents that every employee has a manager. The expected result
 is:
 
-```prolog
-manager(z1,1).
-manager(z2,2).
+```
+manager(y1,1).
+manager(y2,2).
 ```
 
-where `z1` and `z2` are marked nulls, representing that there must be a manager
-for each of the employees, but their identity is unknown.
+where `y1` and `y2` are marked nulls, representing that there must be a manager
+for each of the two employees. We cannot know from this program alone, if they
+are the same person or not, but there exists the possibility that they are
+different people.
 
 ```prolog showLineNumbers
 employee("Jack").
 contract("Jack").
 employee("Ruth").
 contract("Ruth").
 employee("Ann").
-hired("Ann","Ruth").
```

Comment:
You are right: `hired("Ann","Ruth").` is a tuple from the database, so it can't be read by an intentional predicate (The head of the rule `hired(Y,X) :- manager(Y,X), contract(X).`)